capabilities.go

```diff
@@ -139,5 +151,54 @@ func (c *Client) GetCapabilities(target string, port int) (response *Capabilitie
 		err = fmt.Errorf("missing %s version", DefaultServiceName)
 	}
 
+	// Parse PIKE capability
+	if pike, ok := response.Capabilities["935478af7bf2"].(map[string]interface{}); ok {
```

Comment:
We can use `BRFCPike` constant, instead of plain string here.

Reply:
```suggestion
	if pike, ok := response.Capabilities[BRFCPike].(map[string]interface{}); ok {
```